ArchiSteamFarm.Tests/Bot.cs

```diff
@@ -121,6 +122,33 @@ public sealed class Bot {
 			Assert.IsTrue(itemsToSend.Count == 5);
 		}
 
-		private static Steam.Asset CreateCard(ulong classID, uint amount = 1) => new Steam.Asset(Steam.Asset.SteamAppID, Steam.Asset.SteamCommunityContextID, classID, amount, realAppID: 42, type: Steam.Asset.EType.TradingCard, rarity: Steam.Asset.ERarity.Common);
+		[TestMethod]
+		public void SeveralRealAppIDs() {
+			HashSet<Steam.Asset> items = new HashSet<Steam.Asset> {
+				CreateCard(1, realAppID: 42),
+				CreateCard(1, realAppID: 43)
+			};
+
+			try {
+				GetItemsForFullBadge(items, 2);
+				Assert.Fail();
+			} catch (ArgumentException) { }
```

Comment:
This is now how you expect exceptions to happen. Check `[ExpectedException]` attribute.